src/eradiate/_mode.py

```diff
@@ -83,27 +83,55 @@ def _mode_registry() -> dict[str, Mode]:
                 "mi_double_precision": False,
                 "mi_polarized": False,
             },
+            "mono_single_polarized": {
```

Comment:
For consistency with Mitsuba, I'd prefer to change the order: `mono_polarized_single`